src/Components/CommentSection.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {FormEvent, useState} from "react";
+import CommentDialogSection from "./CommentDialogSection";
+import {addComment, getComments} from "../Service/User/RequestsForUsers";
+import {useAppSelector} from "../Hooks/hook";
+import {commentSchema} from "../validator";
+import {ToastNotifyError, ToastNotifySuccess} from "../Helpers";
+import {CommentFromDB, User} from "../types/typeSection";
+import CoreButton from "./common/CoreButton";
+
+interface Props {
+    postId: number
+}
+
+const CommentSection: React.FC<Props> = ({postId}) => {
+    const user:User|null = useAppSelector(state => state.auth.auth)
+    const [loading,setLoading] = useState<boolean>(false)
+    const [open, setOpen] = useState<boolean>(false);
+    const [comment, setComment] = useState<string>('')
+    const token: string = useAppSelector(state => state.auth.token);
+    const [spinIsActive, setSpinIsActive] = useState<boolean>(false)
+    const [postComments, setPostComments] = useState<CommentFromDB[]>([])
+    const handleClickOpen = async (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.preventDefault()
+        console.log(postId)
+            setSpinIsActive(true)
+            const getPostComments = await getComments(postId, token)
+            setPostComments([...getPostComments])
+            setSpinIsActive(false)
+            setOpen(true);
+    };
+    const handleCommentAdding = async (e: FormEvent): Promise<void> => {
+        e.preventDefault()
+        if(user){
+            const validationRules: boolean = await commentSchema.isValid({comment})
+            console.log(validationRules)
+            if (validationRules) {
+                setLoading(true)
+                const commentAdded = await addComment(comment, postId, token)
+                setComment('')
+                setLoading(false)
+                ToastNotifySuccess('Your comment has been added check in show Comments')
+            } else {
+                setLoading(false)
+                ToastNotifyError('Something went wrong (remember text is required and must be lower then 200)🤔')
```

Comment:
not big deal but don't keep this kind of texts in business logic (component method), create some json file from where you can get all of them, also by the way you can integrate i18n for future let's say.